test/studies/hpcc/STREAMS/elliot/stream-spmd-barrier.compopts

```diff
@@ -1 +1,2 @@
-../../common/probSize.chpl
+../../common/probSize.chpl --no-warnings
+
```

Comment:
Extra line? Probably not a big deal, but I'm not 100% sure that start_test will ignore this line (vs. treat it as a second compopts)